The full pharo build fails because of this error, that seems unrelated:
```
Run make -C pharo
  make -C pharo
  shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
  env:
    PROJECT: MachineArithmetic
    Z3_DOWNLOAD_URL_WINDOWS: https://github.com/Z3Prover/z3/releases/download/z3-4.8.17/z3-4.8.17-x64-win.zip
make: Entering directory '/d/a/MachineArithmetic/MachineArithmetic/pharo'
GNUmakefile:3: GNUmakefile.local: No such file or directory
make: Leaving directory '/d/a/MachineArithmetic/MachineArithmetic/pharo'
Error: Process completed with exit code 1.
```

--- a/MachineArithmetic-Tests/ContextlessZ3Test.class.st
+++ b/MachineArithmetic-Tests/ContextlessZ3Test.class.st
@@ -59,10 +59,11 @@ ContextlessZ3Test >> testSymbolFromString [
 { #category : #tests }
 ContextlessZ3Test >> testTwoContexts [
 	"Several logical context can be used simultaneously."
-	| ctx1 ctx2 x y str |
+
+	| ctx1 ctx2 x y |
 	ctx1 := Z3Context fromDefault.
 	ctx2 := Z3Context fromDefault.
-	self deny: ctx1 handle asInteger = ctx2 handle asInteger.
+	self deny: ctx1 getHandle asInteger = ctx2 getHandle asInteger.
 	x := ctx1 mkBoolVar: 'x'.
 	y := ctx2 mkBoolVar: 'y'.
 	ctx1 del.

--- a/MachineArithmetic-Tests/SimpleZ3Test.class.st
+++ b/MachineArithmetic-Tests/SimpleZ3Test.class.st
@@ -42,10 +42,13 @@ SimpleZ3Test >> test0lt1 [
 
 { #category : #tests }
 SimpleZ3Test >> testAllBoolSortHandlesAreSame [
+
 	| boolSort1 boolSort2 |
 	boolSort1 := Z3Sort bool.
 	boolSort2 := Z3Sort bool.
-	self assert: boolSort1 handle value equals: boolSort2 handle value
+	self
+		assert: boolSort1 getHandle value
+		equals: boolSort2 getHandle value
 ]
 
 { #category : #tests }

